### DISCUSSION TITLE
CSharp expression - Access last activity output result object.property

### DISCUSSION THREAD

**Discussion by rmanjusha** [elsa-workflows/elsa-core#5257] [Q&A]
How to access property value of result object from Output.LastResult using Csharp expression

**Comment by glime-ai** [comment]
To access a property value of the result object from `Output.LastResult` using a C# expression in Elsa, you would typically use the dot notation to access the property you are interested in. For example, if `Output.LastResult` returns an object that has a property named `PropertyName`, you would access it like this:

```csharp
Output.LastResult.PropertyName
```

If you need to cast the result to a specific type before accessing the property, you can do so like this:

```csharp
((YourType)Output.LastResult).PropertyName
```

Replace `YourType` with the actual type of the object you are expecting and `PropertyName` with the actual name of the property you want to access.

Please ensure that the property you are trying to access exists on the object returned by `Output.LastResult` and that it is public and accessible.

This comment was generated by [Glime](https://glimelab.ai/?utm_source=dRbvTb7IhCSYiI&utm_medium=github).

**Reply by rmanjusha** [reply]
Tried  ((YourType)Output.LastResult).PropertyName , but getting below error.

error CS0246: The type or namespace name 'MyType' could not be found (are you missing a using directive or an assembly reference?)

How do I add assembly reference ?
